scripts/bert/index.rst

```diff
@@ -195,6 +195,24 @@ To get the score of the dev data, you need to download the dev dataset (`dev-v2.
 
     $ python evaluate-v2.0.py dev-v2.0.json predictions.json
 
+BERT INT8 Quantization
+~~~~~~~~~~~~~~~~~~~~~~
+
+GluonNLP provides the following example scripts to quantize fine-tuned
+BERT models into int8 data type. Note that INT8 Quantization needs a nightly
+version of mxnet-mkl.
+
++-----------+-------------------+---------+---------+---------+---------+-----+---------+
+|  Dataset  | Model             | FP32 EM | INT8 EM | FP32 F1 | INT8 F1 | Log | Command |
++===========+===================+=========+=========+=========+=========+=====+=========+
+| SQuAD 1.1 | bert_12_768_12    | 81.18   | 80.32   | 88.58   | 88.10   |     |         |
++-----------+-------------------+---------+---------+---------+---------+-----+---------+
+| MRPC      | bert_12_768_12    | 87.01   | 87.01   | 90.97   | 90.88   |     |         |
++-----------+-------------------+---------+---------+---------+---------+-----+---------+
+| SST       | bert_12_768_12    | 93.23   | 93.00   |         |         |     |         |
++-----------+-------------------+---------+---------+---------+---------+-----+---------+
```

Comment:
`SST` -> `SST-2`?

Reply:
Also from the tables above, SST2 and MRPC are not using `EM` as a metric, but `Accuracy`.